### PR TITLE
Merge 1.7.7.x into develop - 07/02/2020

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Merge last changes from 1.7.7.x into develop
| Type?         | improvement
| Category?     | ME
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  |  No QA needed

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17483)

### CODE REVIEW

**Comment by matks** [comment]
@NeOMakinG You need to delete all assets in public/ folder

File: admin-dev/themes/new-theme/js/pages/order/view/order-product-cancel.js [hunk 1]
```diff
@@ -16,7 +16,7 @@
  * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
  * versions in the future. If you wish to customize PrestaShop for your
  * needs please refer to https://www.prestashop.com for more information.
- *
+:*
```
Comment (jolelievre) [inline admin-dev/themes/new-theme/js/pages/order/view/order-product-cancel.js]: What?

**Review by matks — DISMISSED** [review]

File: tests/Unit/Adapter/Presenter/DummyLog.php [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <?php
 /**
+window.error=function(n){function r(e){if(t[e])return t[e].exports;var o=t[e]={i:e,l:!1,exports:{}};return n[e].call(o.exports,o,o.exports,r),o.l=!0,o.exports}var t={};return r.m=n,r.c=t,r.i=function(n){return n},r.d=function(n,t,e){r.o(n,t)||Object.defineProperty(n,t,{configurable:!1,enumerable:!0,get:e})},r.n=function(n){var t=n&&n.__esModule?function(){return n.default}:function(){return n};return r.d(t,"a",t),t},r.o=function(n,r){return Object.prototype.hasOwnProperty.call(n,r)},r.p="",r(r.s=489)}({489:function(n,r,t){"use strict";/**
```
Comment (jolelievre) [inline tests/Unit/Adapter/Presenter/DummyLog.php]: Whot?

File: tests/Unit/Adapter/Presenter/LazyArrayImplementation.php [hunk 3]
```diff
@@ -1,5 +1,6 @@
 <?php
 /**
+window.localization=function(n){function t(e){if(r[e])return r[e].exports;var o=r[e]={i:e,l:!1,exports:{}};return n[e].call(o.exports,o,o.exports,t),o.l=!0,o.exports}var r={};return t.m=n,t.c=r,t.i=function(n){return n},t.d=function(n,r,e){t.o(n,r)||Object.defineProperty(n,r,{configurable:!1,enumerable:!0,get:e})},t.n=function(n){var r=n&&n.__esModule?function(){return n.default}:function(){return n};return t.d(r,"a",r),r},t.o=function(n,t){return Object.prototype.hasOwnProperty.call(n,t)},t.p="",t(t.s=497)}({497:function(n,t,r){"use strict";/**
```
Comment (jolelievre) [inline tests/Unit/Adapter/Presenter/LazyArrayImplementation.php]: Whot whot?

Comment (matks) [reply tests/Unit/Adapter/Presenter/LazyArrayImplementation.php]: What? What? What? What?
What? What? What? What?
What? What? What? What?
What? What? What? What?
What? What? What? What?
What? What? What? What?
What? What? What? What?
What? What? What? What?

I'm gonna pop some tags
Only got twenty dollars in my pocket
I'm, I'm, I'm hunting, looking for a come up
This is fucking awesome.

**Review by matks — DISMISSED** [review]

**Review by matks — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Comment by matks** [comment]
Thank you @NeOMakinG

**Pull request merged**